src/SSHDebugPS/Docker/DockerHelper.cs

```diff
@@ -17,6 +17,123 @@ public class DockerHelper
         private const string dockerPSCommand = "ps";
         // --no-trunc avoids parameter truncation
         private const string dockerPSArgs = "-f status=running --no-trunc --format \"{{json .}}\"";
+        private const string dockerInfoCommand = "info";
+        private const string dockerInfoArgs = "-f {{.Driver}}";
+        private const string dockerVersionCommand = "version";
+        private const string dockerVersionArgs = "-f {{.Server.Os}}";
+        private const string dockerInspectCommand = "inspect";
+        private const string dockerInspectArgs = "-f \"{{json .Platform}}\" ";
+        private static char[] charsToTrim = { ' ', '\"' };
+
+        public static bool LCOW(string hostname, out bool lcow)
+        {
+            string driverInfo = string.Empty;
+            bool temp = false;
+            DockerCommandSettings settings = new DockerCommandSettings(hostname, false);
+            settings.SetCommand(dockerInfoCommand, dockerInfoArgs);
+            bool status = TryGetDockerCommandOutput(settings, out driverInfo);
+            if (driverInfo.Contains("lcow"))
```

Comment:
`private const string LCOW = "lcow"`

Remove temp and simplify to `lcow = driverInfo.Contains(LCOW)`